src/components/Header/BurgerMenu.jsx

```diff
@@ -0,0 +1,19 @@
+import { slide as Menu } from 'react-burger-menu'
+import React from 'react';
+import { Link} from 'react-router-dom';
+
+const BurgerMenu = ()=>{
```

Comment:
I'm going to approve this as the main functionality is good, but moving forward please make sure not to include any files that aren't relevant to the ticket in your commits.